transport.go

```diff
@@ -62,3 +65,12 @@ func (t DMSGTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 
 	return http.ReadResponse(bufio.NewReader(conn), req)
 }
+
+var once sync.Once
```

Comment:
If we create another `DMSGTransport`with this, we won't be able to instantiate another `dmsg.Client`. I recommend putting this as a member of `DMSGTransport`.

Reply:
True that, forgot to move it after client was moved from a package to the DMSGTransport instance